modules/pal/utils/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why need to put into `modules/`?

Reply:
I want to be isolated from code that is not modularized

All scripts that have not been modularized are placed outside the modules folder

Reply:
You can just record it in other way, but not use folder to do it.

Reply:
we also config the module workspaces here, the module declare in these directory could be linked while we execute `npm i` command
https://github.com/cocos/cocos-engine/blob/6b3fa742382acfe3b2e8dd5e5712a5594cd9c1d5/package.json#L15-L18

Reply:
I think they still needs to be put in a separate folder, because each module is an independent ts project, they have their own `tsconfig.json`, which is no longer suitable for putting together with the other scripts

Reply:
Can just use pal folder. Maybe there is pal/config folder to record configuration files. It is strange to see
```
modules/a
modules/b
```
in engine. Because a and b should be a module.

Reply:
the`/modules/pal` can be merged into `/pal` after we modularize all pal modules

Reply:
It seems it's OK to merge into pal directory
https://github.com/cocos/cocos-engine/pull/16196/commits/9dc9cf5feba2745028cef6def66597b5e631669a